pkg/controller/networkpolicy/endpoint_querier_test.go

```diff
@@ -12,21 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package networkpolicy
+package networkpolicy_test
 
 import (
+	"fmt"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
 	corev1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 
-	"antrea.io/antrea/pkg/apis/controlplane/v1beta2"
+	"antrea.io/antrea/pkg/apis/controlplane"
+	crdv1beta1 "antrea.io/antrea/pkg/apis/crd/v1beta1"
+	"antrea.io/antrea/pkg/controller/networkpolicy"
```

Comment:
I renamed the this package to `networkpolicy_test` for a work around, because it need to use mock defined in networkpolicy_testing. Otherwise, there will be an import cycle not allowed.

Reply:
Normally the testing package is created for other packages that consume interfaces defined in this package so even it imports the parent package the parent doesn't import it. But we have too many objects in this package and end up with mocking an interface when testing another. In the future we should consider divide the package.

For now I think the type EndpointNetworkPolicyRules could be move to another package like `pkg/controller/types`.